src/server/frontend_wayland/wlr_screencopy_v1.cpp

```diff
@@ -156,7 +137,8 @@ void mf::WlrScreencopyManagerV1::capture_output(
     struct wl_resource* output)
 {
     (void)overlay_cursor;
-    new WlrScreencopyFrameV1{frame, ctx, ctx->extents_of_output(output)};
+    auto const extents = OutputGlobal::from_or_throw(output).current_config().extents();
```

Comment:
Hm. Failure here should ideally result in a `wl_display::invalid_object` protocol error, right?

Huh. Unfortunately we can't send that ourselves!

Hm. Actually, I *think* libwayland guarantees that we get a valid resource, so the only question here is whether the resource is actually a `wl_output`?

I think the takeaway here is that what you have here is fine?